code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -120,3 +120,11 @@
 	volume = 60
 	amount_per_transfer_from_this = 30
 	list_reagents = list("atropine" = 10, "epinephrine" = 10, "salbutamol" = 20, "spaceacillin" = 20)
+
+/obj/item/weapon/reagent_containers/hypospray/medipen/survival
+	name = "Survival Medipen"

```

Comment:
that same logic could justify removing the brute medkit.

Convenience should be a good incentive for mining points. It's pretty damn pricey for a one off heal. No miner is realistically going to go to chem and wait 5 minutes while they make a good good patch for them, it's a huge drop in efficiency.